auth0/v3/authentication/database.py

```diff
@@ -36,18 +36,38 @@ def login(self, client_id, username, password, connection, id_token=None,
             headers={'Content-Type': 'application/json'}
         )
 
-    def signup(self, client_id, email, password, connection):
+    def signup(self, client_id, email, password, connection, username=None,
+             user_metadata=None):
         """Signup using username and password.
```

Comment:
Consider "email and password" since username is optional

Reply:
Would also suggest mentioning here that this is DB connections only

Reply:
I will skip (2) since it's already explained at the top of the file/class.

Reply:
I still think it would be helpful to add it here like:

> Sign a user up with a database connection using username and password.

... or similar. I'll approve regardless, minor thing.